src/Symfony/Contracts/HttpClient/Test/HttpClientTestCase.php

```diff
@@ -948,4 +949,40 @@ public function testMaxDuration()
 
         $this->assertLessThan(10, $duration);
     }
+
+    public function testResponseStreamRewind()
```

Comment:
toStream is not in the contracts, this should be added to the test class in the component instead. (that should fix test failures)

Reply:
Test is still failing on Travis (AppVeyor failure seems unrelated). Should I move the other 2 tests to the component?

Reply:
my comment applied to all added test cases :)

Reply:
ah, all good now :)